src/test/java/igrad/logic/LogicManagerTest.java

```diff
@@ -1,14 +1,9 @@
-package seedu.address.logic;
+package igrad.logic;
 
+import static igrad.commons.core.Messages.MESSAGE_INVALID_MODULE_DISPLAYED_INDEX;
+import static igrad.commons.core.Messages.MESSAGE_UNKNOWN_COMMAND;
+import static igrad.testutil.Assert.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static seedu.address.commons.core.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
-import static seedu.address.commons.core.Messages.MESSAGE_UNKNOWN_COMMAND;
-import static seedu.address.logic.commands.CommandTestUtil.ADDRESS_DESC_AMY;
-import static seedu.address.logic.commands.CommandTestUtil.EMAIL_DESC_AMY;
-import static seedu.address.logic.commands.CommandTestUtil.NAME_DESC_AMY;
```

Comment:
please retain static imports (instead of removing them), they're much cleaner. i'll prob change it back then

Reply:
Yes thank you!